packages/next/test/unit/fixtures/01-normalize-paths/index.test.js

```diff
@@ -38,9 +39,15 @@ describe(`${__dirname.split(path.sep).pop()}`, () => {
 
     for (const page of pages) {
       expect(output).toHaveProperty(page);
-      expect(path.resolve(output[page].fsPath)).toEqual(
-        path.join(pagesDir, `${page}.html`)
-      );
+      if (page === 'index') {
+        const { files, type } = output[page];
+        expect(type).toEqual('Lambda');
+        expect(files).toHaveProperty([path.join('data', 'strings.json')]);
+      } else {
+        expect(path.resolve(output[page].fsPath)).toEqual(
+          path.join(pagesDir, `${page}.html`)
+        );
+      }
```

Comment:
```suggestion
      expect(path.resolve(output[page].fsPath)).toEqual(
        path.join(pagesDir, `${page}.html`)
      );
```

Remove the `else`? Seems like this should always be true

Reply:
I don't think will work. When `page` is `index`, `output[page]` is a `NodejsLambda` instance which does not have a `fsPath` property.

Reply:
Gave it a try and indeed `fsPath` is undefined:

```
    TypeError: The "path" argument must be of type string. Received undefined

      45 |         expect(files).toHaveProperty([path.join('data', 'strings.json')]);
      46 |       }
    > 47 |       expect(path.resolve(output[page].fsPath)).toEqual(
         |                   ^
      48 |         path.join(pagesDir, `${page}.html`)
      49 |       );
      50 |     }

      at Object.<anonymous> (test/unit/fixtures/01-normalize-paths/index.test.js:47:19)
```